addon/utils/read-modules.js

```diff
@@ -51,7 +50,7 @@ export default function(prefix) {
 
     let data = module.default;
 
-    modulesMap[moduleType][_camelCase(moduleKey)] = data;
+    modulesMap[moduleType][camelize(moduleKey)] = data;
```

Comment:
this is needed to match the name while we lookup factories https://github.com/samselikoff/ember-cli-mirage/blob/f0e5c30add1117b673014d4918abfa087b273019/addon/server.js#L338-L339